fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -614,9 +656,9 @@ struct LogGenerationData : NonCopyable, public ReferenceCounted<LogGenerationDat
 	                           UID recruitmentID,
 	                           ProtocolVersion protocolVersion,
 	                           TLogSpillType logSpillType,
-	                           std::unordered_map<ptxn::StorageTeamID, std::vector<Tag>>& storageTeams,
+	                           std::map<ptxn::StorageTeamID, std::vector<Tag>>& storageTeams,
 	                           int8_t locality,
-	                           DBRecoveryCount epoch,
+	                           //    UID epoch,
```

Comment:
We probably don't want to merge code with commented out argument. Why not removing it if not needed?

Reply:
it seems no one knows whether we need it, so i just posted here in case people find we need it, I can certainly remove it.